internal/taskfile/read/taskfile.go

```diff
@@ -44,6 +45,24 @@ func Taskfile(dir string, entrypoint string) (*taskfile.Taskfile, error) {
 		if len(includedTaskfile.Includes) > 0 {
 			return nil, ErrIncludedTaskfilesCantHaveIncludes
 		}
+
+		includedTaskDirectory := filepath.Dir(path)
+		includedTaskfileName := strings.TrimSuffix(filepath.Base(path), filepath.Ext(path))
+		path = filepath.Join(includedTaskDirectory, fmt.Sprintf("%s_%s.yml", includedTaskfileName, runtime.GOOS))
+		if _, err = os.Stat(path); err == nil {
+			osIncludedTaskfile, err := readTaskfile(path)
+			if err != nil {
+				return nil, err
+			}
+			if err = taskfile.Merge(includedTaskfile, osIncludedTaskfile); err != nil {
+				return nil, err
+			}
+		}
+
+		for _, task := range includedTaskfile.Tasks {
+			task.Dir = filepath.Join(includedTask.Dir, task.Dir)
```

Comment:
We need to skip joining if the path in the task is absolute:

```go
if !filepath.IsAbs(task.Dir) {
	task.Dir = filepath.Join(includedTask.Dir, task.Dir)
}
```

Reply:
You're right, also I fixed this point for included files (`internal\taskfile\read\taskfile.go:32`), it works incorrectly in some cases.
Example:
![image](https://user-images.githubusercontent.com/10252305/74590302-39f7bf00-501e-11ea-9467-0418626c1159.png)